administrator/components/com_cpanel/Controller/SystemController.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_cpanel
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace Joomla\Component\Cpanel\Administrator\Controller;
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\Factory;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\MVC\Controller\BaseController;
+use Joomla\CMS\Response\JsonResponse;
+use Joomla\CMS\Updater\Updater;
+
+/**
+ * Cpanel System Controller
+ *
+ * @since  1.5
```

Comment:
You can time travel? :D 
```suggestion
 * @since  __DEPLOY_VERSION__
```